requirements.txt

```diff
@@ -32,5 +32,6 @@ Sphinx==1.7.1
 sphinx-rtd-theme==0.2.4
 sphinxcontrib-websupport==1.0.1
 typing==3.6.4
+unidecode==1.0.22
```

Comment:
you have to add the dependency to the `setup.py` file too ;)

Reply:
Done